lib/karafka/app.rb

```diff
@@ -15,7 +14,7 @@ class << self
       def boot!
         Setup::Config.validate!
         Setup::Config.setup_components
-        Callbacks.after_init(Karafka::App.config)
+        events.publish('app.after_init', config: Karafka::App.config)
```

Comment:
we should have a single place where all the system events are defined (apart from the dynamic per controller once). This makes code easier to read for users.

Reply:
Again not sure what you mean here. They are defined in the `EventPublisher`. We could move the event names to constants defined in a separate module, but since you requested that dry-events should raise an error when publishing an unregistered event, I'm not sure we need to.

Reply:
Dunno why but during the review I completely missed this: https://github.com/karafka/karafka/pull/305/files#diff-9956e073809cb05c57c6dd8df8777aeeR8 - everything is fine here apart from moving it to a constant just like events for the instrumentation: https://github.com/karafka/karafka/blob/master/lib/karafka/instrumentation/monitor.rb#L23